test/gui/shared/scripts/helpers/api/Provisioning.py

```diff
@@ -4,6 +4,8 @@
 from helpers.api.utils import url_join
 from helpers.ConfigHelper import get_config
 
+created_groups = {}
```

Comment:
```suggestion
created_groups = []
```

Reply:
but it might not work with ocis. because we delete group using group-id
So may be have a store like this:
```py
created_groups = {
  "<groupname>": {
      "id": <id>
      "name": <groupname>
    }
}
```

Reply:
i think we can directly add group_name in id, since I saw group_name being used as groupid

Reply:
for oc10 case: yes but for ocis it will be confusing